catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -73,6 +73,7 @@
                     <value>lon</value>
                     <value>radius</value>
                     <value>bbox</value>
+                    <value>geometry</value>
```

Comment:
❗️ The documentation also needs to be updated: https://github.com/codice/ddf/blob/43d0aff293e1a9c2f60447cd409013654f84540a/distribution/docs/src/main/resources/content/_tables/OpenSearchSource.adoc#L47-L52